packages/common-cli-options-help/src/index.ts

```diff
@@ -41,7 +41,7 @@ export const UNIVERSAL_OPTIONS = [
     shortAlias: '-C',
   },
   {
-    description: 'What level of logs to report. Any logs of higher level than the setting are shown. Values: silent, error, warn, info, debug. Default: info',
+    description: 'What level of logs to report. Any logs at or higher than the given level will be shown. Levels (lowest to highest): debug, info, warn, error. Or use "--silent" to turn off all logging',
```

Comment:
Our convention from previous commits was to leave out the final period for a single sentence, but include the final period when there are multiple sentences. Otherwise the final sentence seems to be left hanging.